### PR TITLE
Update configs and CRDs in how to guide docs

### PR DESCRIPTION
**Pull request by Jayanie**
This PR includes the following changes to how to guide docs:

- how to configure the default security modifications
- update the apply rate limiting and security docs with related to defining the namespace

### CODE REVIEW

File: docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ This policy cofigmap is mounted to kaniko job by the API controller and used to
 Create a ratelimiting policy using the RateLimiting kind. You can create 3 types of policies with the RateLimiting kind as shown below.
 Include the limit from which you want to throttle the API requests.
 
+Note: When an API refers a ratelimiting policy in swagger definition under x-wso2-throttling-tier keyword you need to make sure that the namespace that you have
+provided the ratelimiting policy is same as the namespace of the API. 
+
+Example: In the following ratelimiting policies the namespace is provided as "wso2-test-ns". Therefore the namespace of the API that these ratelimiting policies are being
+referred should be "wso2-test-ns".
```
Comment (Meruja) [inline docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md:21]: Shall we rephrase it as below? (can add line 21 with 20)
In the following ratelimiting policies, the namespace is provided as "wso2-test-ns". Therefore the namespace of the API that these ratelimiting policies are being referred should be "wso2-test-ns".

File: docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ This policy cofigmap is mounted to kaniko job by the API controller and used to
 Create a ratelimiting policy using the RateLimiting kind. You can create 3 types of policies with the RateLimiting kind as shown below.
 Include the limit from which you want to throttle the API requests.
 
+Note: When an API refers a ratelimiting policy in swagger definition under x-wso2-throttling-tier keyword you need to make sure that the namespace that you have
+provided the ratelimiting policy is same as the namespace of the API.
```
Comment (tgtshanika) [inline docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md:18]: 1. `,` is required after "...under x-wso2-throttling-tier keyword" -> ...under x-wso2-throttling-tier keyword,
2. Reword as the namespace that you have
`specified for` the ratelimiting policy is same as the namespace of the API

File: docs/HowToGuide/configurations-overview.md [hunk 3]
```diff
@@ -59,7 +59,11 @@
 
 - API Operator provides the security to your API. You need to define the security which needs to be applied on the API under "security" extension in the API.
 
-- But if you have not specified the security in the API definition, the API operator will apply the default security for the API.
+- At the time of the installation of the API Operator the default security will be applied. If you need to change the default security configurations either you can edit the security custom resource or you can edit the file within the distribution.
```
Comment (tgtshanika) [inline docs/HowToGuide/configurations-overview.md:62]: 1. `,` is required after the ...installation of the API Operator 
2. `,` is required after the ...If you need to change the default security configurations

File: docs/HowToGuide/configurations-overview.md [hunk 4]
```diff
@@ -59,7 +59,11 @@
...
+
+- If you need to apply the same security configurations to all the APIs then you need to change the default security configurations. If you prefer any other security type as the default security type, you may need to change the values of the default security configuration.
```
Comment (tgtshanika) [inline docs/HowToGuide/configurations-overview.md:64]: Break the sentence by a comma in `If you need to apply the same security configurations to all the APIs `

File: docs/HowToGuide/configurations-overview.md [hunk 5]
```diff
@@ -88,7 +92,11 @@
...
     type: Opaque
      ```
-
+    - If you have changed the default security custom resource then you need to re apply it after doing the modifications using the following command
```
Comment (tgtshanika) [inline docs/HowToGuide/configurations-overview.md:95]: `,` required before `then`

File: docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md (hunk 1 above)
Comment (Jayanie) [reply docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md:21]: Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03

File: docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md (hunk 2 above)
Comment (Jayanie) [reply docs/HowToGuide/OverviewOfCrds/apply-rate-limiting-to-api.md:18]: Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03

File: docs/HowToGuide/configurations-overview.md (hunk 3 above)
Comment (Jayanie) [reply docs/HowToGuide/configurations-overview.md:62]: Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03

File: docs/HowToGuide/configurations-overview.md (hunk 4 above)
Comment (Jayanie) [reply docs/HowToGuide/configurations-overview.md:64]: Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03

File: docs/HowToGuide/configurations-overview.md (hunk 5 above)
Comment (Jayanie) [reply docs/HowToGuide/configurations-overview.md:95]: Fixed with c46ba03a4a619b503842bcdbed458e8be91c4b03

**Pull request merged**